console/src/cluster/Dropdown.tsx

```diff
@@ -163,12 +185,19 @@ const ListItem = (props: CoreList.ItemProps<string, Cluster>): ReactElement => {
         break;
     }
   }
+
+  const handleChange = (value: string) => {
+    if (value.length === 0) return;
```

Comment:
It shouldn't be possible for this value to be of length 0, `Text.Editable` prevents it. Can we remove?